framework/src/base/NonlinearSystem.C

```diff
@@ -1786,16 +1795,65 @@ NonlinearSystem::computeJacobianInternal(SparseMatrix<Number> &  jacobian)
       {
         _fe_problem.reinitNodeFace(node, boundary_id, 0);
```

Comment:
I thought that `boundary_info()` was created based on sidesets and nodesets. Then, just because there is a sideset, it does not mean there will be a MOOSE BC object on it. But, if the first sentence is not true, then disregard...